.github/workflows/turf.yml

```diff
@@ -18,13 +18,20 @@ jobs:
         node-version: [16.x, 18.x, 20.x]
 
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
```suggestion
        uses: actions/checkout@v4
```

Reply:
context: github actions use node and I think checkout v4 uses node 20

Reply:
Will give the newer version a whirl.